src/Microsoft.DotNet.Interactive/ChooseKeyValueStoreKernelDirective.cs

```diff
@@ -46,7 +46,7 @@ public class ChooseKeyValueStoreKernelDirective : ChooseKernelDirective
 
                 if (!File.Exists(filePath))
                 {
-                    result.ErrorMessage = LocalizationResources.Instance.FileDoesNotExist(filePath);
+                    result.ErrorMessage = System.CommandLine.LocalizationResources.Instance.FileDoesNotExist(filePath);
```

Comment:
This API will be internal in the next version so we should create our own resource for it in this project.

Reply:
resource added to avoid using System.CommandLine.LocalizationResources